src/client/serialize/daos_serialize.c

```diff
@@ -857,17 +850,17 @@ deserialize_props(daos_handle_t poh, hid_t file_id, daos_prop_t **_prop,
 	}
 	/* deserialize_label stays false if property doesn't exist above */
 	if (deserialize_label) {
-		type = DAOS_PROP_CO_LABEL;
-		prop->dpp_entries[prop_num].dpe_type = type;
-		prop->dpp_entries[prop_num].dpe_str =
-						strdup(label_entry->dpe_str);
+		prop->dpp_entries[prop_num].dpe_type = DAOS_PROP_CO_LABEL;
```

Comment:
@mchaarawi does the DAOS_PROP_CO_ACL also need to be updated to use daos_prop_entry_set_ptr since it uses a dpe_val_ptr?

Reply:
no it's already using an API (daos_acl_from_strs) to create the ACL.